.travis.yml

```diff
@@ -46,11 +46,14 @@ deploy:
 jobs:
   include:
     - stage: Build & Bundle
-      env: ROS_DISTRO=dashing ROS_VERSION=2 GAZEBO_VERSION=9 WORKSPACES="robot_ws"
+      env: ROS_DISTRO=dashing ROS_VERSION=2 GAZEBO_VERSION=9 WORKSPACES="robot_ws" UPLOAD_SOURCES="false"
```

Comment:
Related to https://github.com/aws-robotics/travis-scripts/pull/60#discussion_r331276402, `UPLOAD_SOURCES` should just always be `false` in the context of `common_sa_build.sh`, so declaring it here is unnecessary.

Reply:
What do you mean? the default behavior is to upload sources as part of the build jobs, so I'm disabling it explicitly for this one.

Reply:
Never mind. `UPLOAD_SOURCES` is effectively `true` by default when undefined for backwards compatibility. Earlier I forgot about the backwards compatibility part.